plugins/k8sauditeks/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2022 The Falco Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+SHELL=/bin/bash -o pipefail
+GO ?= go
+CURL ?= curl
+
+FALCOSECURITY_FALCO_REPO ?= falcosecurity/falco
+FALCOSECURITY_FALCO_REVISION ?= 65435d4418638176f68df9017df36cd5d94ba083
+
+TEST_FILES_DIR := test_files
+NAME := k8sauditeks
+OUTPUT := lib$(NAME).so
+
+ifeq ($(DEBUG), 1)
+    GODEBUGFLAGS= GODEBUG=cgocheck=2
+else
+    GODEBUGFLAGS= GODEBUG=cgocheck=0
+endif
+
+all: $(OUTPUT)
+
+clean:
+	@rm -f *.so *.h
+
+test-files:
```

Comment:
This is a copy-paste from k8saudit I guess! I think we should not include this for now to reduce entropy. This was put there just to provide a barebones benchmark attempt, but we need a better testing strategy in the general case.

Reply:
I'll clean that.